lib/payment_processor/clients/braintree/subscription_plan_selector.rb

```diff
@@ -3,6 +3,8 @@ module Clients
     module Braintree
 
       class SubscriptionPlanSelector
+        include ActsLikeSelectorWithCurrency
+
         # I know, I know! This is a temporary measure.
```

Comment:
I took them out :grimacing:. I just couldn't bring myself to insert all those ENV values on staging.